### DISCUSSION TITLE
Display message field in table

### DISCUSSION THREAD

**Discussion by marcohald** [fbaligand/kibana-enhanced-table#347] [Q&A]
Hi,

I want to build an enhanced Table with the message field as column.
My Problem is that the message field is not displayed in the Terms Field.
![image](https://github.com/fbaligand/kibana-enhanced-table/assets/10107699/0eb3fbc0-3885-411d-9969-e8caa9203bc4)
I'm however not sure if this is the right use case for this Plugin, maybe a saved search would be better but as far as i know I could not set a column width in Opensearch Dashboards.

Thanks

**Comment by fbaligand** [comment]
By default, terms query works only with keyword typed fields. I guess that your message field is typed “text”. That’s why you don’t see this field in the field list. 

If you want one table line = one document, then yes Saved Search is the good viz, given that you don’t need aggregation. 
By the way, you can also use Document Table visualization (that is part of the plugin), it has the same features than Enhanced Table, but to display one document per table line (as Saved Search).
